tests/test-options.ts

```diff
@@ -0,0 +1,98 @@
+import { test as base } from '@playwright/test';
+import { PageManager } from '../pages/pageManager';
+import { RandomDataHelper } from '../pages/randomDataHelper'
+import { APIHelper} from '../pages/apiHelper'
+
+
+export type TestOptions = {
+    pageManager: PageManager
+    newOwnerFixture: [{
+        'firstName': string,
+        'lastName': string,
+        'address': string,
+        'city': string,
+        'telephone': string,
+        'petName': string,
+        'petBirthdate': Date,
+        'petType': string,
+        'visitDate': Date,
+        'visitDescription': string
+    }]
+}
+
+function setOwnerPetAndVisitVariables(){
+    const randomDataHelper = new RandomDataHelper()
+    const testVars = {
+        'firstName': randomDataHelper.getFirstName(),
+        'lastName': randomDataHelper.getLastName(),
+        'address': randomDataHelper.getAddress(),
+        'city': randomDataHelper.getCity(),
+        'telephone': randomDataHelper.getPhone(),
+        'petName': randomDataHelper.getPetName(),
+        'petBirthdate': randomDataHelper.getPetBirthdate(),
+        'petType': randomDataHelper.getPresetPetType(),
+        'visitDate': new Date(),
+        'visitDescription': randomDataHelper.getVisitDescription()
+
+    }
+    return testVars
```

Comment:
This function is not needed. You can create this object inside of the fixture method and the read the values from the object. So that way you don't need to recreate all those constants again in the fixture method newOwnerFixture.

Reply:
This makes sense.  That function morphed out of an earlier idea, when I was trying to figure out how to pass data in or out of the fixture.